be/src/common/daemon.cpp

```diff
@@ -322,15 +322,18 @@ void Daemon::memory_maintenance_thread() {
         doris::ExecEnv::GetInstance()->workload_group_mgr()->do_sweep();
         doris::ExecEnv::GetInstance()->workload_group_mgr()->refresh_wg_weighted_memory_limit();
 
-        // step 7. Analyze blocking queries.
+        // step 7: handle paused queries(caused by memory insufficient)
+        doris::ExecEnv::GetInstance()->workload_group_mgr()->handle_paused_queries();
+
+        // step 8. Analyze blocking queries.
```

Comment:
Remove [step 8. Analyze blocking queries.]
The TODO [step 8. Analyze blocking queries.] here and the newly added [step 7: handle paused queries (caused by insufficient memory)] have the same meaning.